src/serlo-editor/plugins/image/renderer.tsx

```diff
@@ -33,7 +33,7 @@ export function ImageRenderer({
         {wrapWithLink(
           placeholder ?? (
             <img
-              className="serlo-img"
+              className="serlo-img serlo-img-error-placeholder"
```

Comment:
you can add tailwind directly here.

Reply:
@hejtful I'm pretty strict about adding editor stuff into the renderer. The renderer should remain as "pure" as possible.
What we need in the long term is a smarter Image component that parses the url, tries to load the image etc…
In the short term something like that would be okay, but maybe you could add a comment about that.

Reply:
@hejtful adding it in this div here should also work maybe?
https://github.com/serlo/frontend/blob/c3fb04da40b1b16e6a61c41dc1ed69b1451626df/src/serlo-editor/plugins/image/editor.tsx#L57

Reply:
> adding it in this div here should also work maybe?

I couldn't make this work, unfortunately. Added a comment, though.